mods/braindance_protocol/cheats/teleport.lua

```diff
@@ -48,14 +48,14 @@ function Teleport.GenericTotentanzClub()
     Utilities.FinishProtocol(moduleName)
 end
 
-function Teleport.GenericDMCInside()
+function Teleport.GenericDeMCInside()
```

Comment:
Is this a typo? Why `DeMC` instead of `DMC` if it is for `Dark Matter Club` ?

Reply:
Oh, whoops. Should've been DaMC or DMC. I'll fix that 👍